packages/perseus/src/widgets/interactive-graphs/grid.tsx

```diff
@@ -0,0 +1,29 @@
+import {Coordinates} from "mafs";
+import * as React from "react";
+
+import type {PerseusImageBackground} from "../../perseus-types";
+import type {SizeClass} from "../../util/sizing-utils";
+
+const renderLineLabel = (n: number, [min, max]: [number, number]) =>
+    n !== -1 && n !== min && n !== max;
+
+export const Grid = (props: {
+    step: number[];
+    gridStep: number[];
```

Comment:
Can these be `[number, number]` instead (to make sure there's only ever 2 numbers)?